specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -655,7 +655,7 @@
           "Test"
         ],
         "summary": "List server metrics configuration for the given test.",
-        "operationId": "Test_ListServerMetricsConfig",
+        "operationId": "Test_GetTestServerMetricsConfig",
```

Comment:
similarly, I think this should be, `Test_GetTestServerMetricsConfig`

Reply:
Should we keep it `Test_GetServerMetricsConfig`? Since client name already has Test or TestRun

Reply:
changed to LoadTestAdministration_CreateOrUpdateServerMetricsConfig